src/util.ts

```diff
@@ -118,7 +113,20 @@ export const badgeConstants = {
     defaultCertExpireDownDays: "7"
 };
 
-/** Flip the status of s */
+export const durationUnits = {
```

Comment:
Don't think we need to extract this.
Also I think in v1.0 we did not have `m` be minutes, but would need to look this up